hedera-mirror-importer/src/main/resources/db/migration/v1/V1.39.0__support_nfts.sql

```diff
@@ -0,0 +1,44 @@
+-------------------
+-- Update and add tables for NFT support
+-------------------
+
+-- Update the token table
+alter table token
+    add column max_supply bigint not null default 9223372036854775807, -- max long
+    add column supply_type character varying(10) not null default 'INFINITE',
+    add column type character varying(20) not null default 'NON_FUNGIBLE_UNIQUE';
```

Comment:
I believe the default should be `FUNGIBLE_COMMON`
```suggestion
    add column type character varying(20) not null default 'FUNGIBLE_COMMON';
```

Reply:
Correct, not sure why I got that swapped.  Fixed.